LICENCE

```diff
@@ -1,6 +1,4 @@
-The MIT License (MIT)
-
-Copyright (c) 2015 GOV.UK
+Copyright (C) 2015 Crown Copyright (Government Digital Service)
```

Comment:
Think the `(C)` should be `(c)` here (according to usage across other GDS projects).